masterlist.yaml

```
@@ -14838,14 +14846,15 @@ plugins:
         condition: 'version("OCW_Obscure''s_CollegeofWinterhold.esp", "1.5.00", <)'
       - name: 'WACCF_Armor and Clothing Extension.esp'
         condition: 'version("OCW_Obscure''s_CollegeofWinterhold.esp", "1.5.00", <)'
+      - name: 'CollegeOfWinterholdImmersive.esp'
+        condition: 'version("OCW_Obscure''s_CollegeofWinterhold.esp", "1.5.00", <)'
```

Should probably only apply the after if this patch is being used.
Someone else could release another patch with a different order.
The version might not be needed as well, the `<`means it would only work for version prior to `1.5.00`.
```suggestion
        condition: 'active("UltimateCollege.esp")'
```